de/nmichael/efa/data/storage/DataAccess.java

```diff
@@ -34,50 +33,46 @@
     protected String storagePassword;
     protected String storageObjectVersion;
 
-    protected final LinkedHashMap<String,Integer> fieldTypes = new LinkedHashMap<String,Integer>();
+    protected final LinkedHashMap<String, Integer> fieldTypes = new LinkedHashMap<String, Integer>();
```

Comment:
@tfyh Nit: Not that important, but please locally check your changes ("git diff") to make sure what you're pushing is only real changes. Please avoid making formatting changes to existing code to make it easier to review the code. Ideally you'd notice those accidental formatting changes locally and revert them before pushing your code.